server/graphql_schemas/utils/helpers.py

```diff
@@ -178,3 +180,16 @@ def upload_image_file(uploaded_file):
         url = "http://localhost:8000/static{}".format(
             fs.url(filename))
         return url
+
+
+async def send_bulk_update_message(event_instance, message, notification_text):
```

Comment:
Great job on this implementation @Alameen688. Is it possible to debounce to sending of each message by 1 second at least? The `chat.postMessage` method is rate-limited by 1 message per second on Slack. Please see: https://api.slack.com/docs/rate-limits#tier_t5

Reply:
Thanks for pointing that out. I'm on it